ush/global_ens/plot_util.py

```diff
@@ -334,12 +384,16 @@ def get_stat_file_line_type_columns(logger, met_version, line_type):
             'SPEED_ABSERR_NCU', 'DIR_ERR', 'DIR_ERR_NCL', 'DIR_ERR_NCU',
             'DIR_ABSERR', 'DIR_ABSERR_NCL', 'DIR_ABSERR_NCU'
          ]
-      else:
-         logger.error("FATAL ERROR: VCNT is not a valid LINE_TYPE in METV"+met_version)
-         sys.exit(1)
+       else:
+          logger.error("FATAL ERROR: VCNT is not a valid LINE_TYPE in METV"+met_version)
+          exit(1)
```

Comment:
Just wondering if there is a particular reason why `sys.exit(1)` was replaced with `exit(1)`.

Reply:
I saw exit(1) is used in [Marcel's code](https://github.com/NOAA-EMC/EVS/blob/36bbecfda138b0a8ab1827e8fa1526efc97fb499/ush/cam/plot_util.py#L407), so I updated it to be the same.